JSDemos/Demos/TreeView/TreeViewWithSearchBar/description.md

```diff
@@ -1 +1,14 @@
-This demo demonstrates the TreeView component&rsquo;s searching capabilities. The component can find nodes containing your search string or&nbsp;only starting with it&nbsp;depending on&nbsp;the search mode.
\ No newline at end of file
+To add a search bar to the TreeView, set the [searchEnabled](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#searchEnabled) property to **true**.
+
+### Configure search properties
+* Search mode         
+To appear in the search results, a node's text should contain the search string. Use the [searchMode](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#searchMode) property to specify whether the text should start with or be equal to the search string instead. In this demo, you can use the drop-down list to switch between these modes.
+
+* Search fields             
+The TreeView searches through the [text](/Documentation/ApiReference/UI_Components/dxTreeView/Node/#text) node fields. To specify different field(s), use the [searchExpr](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#searchExpr) property.
+
+Note that only loaded nodes appear in the search results. If the TreeView uses the [createChildren](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#createChildren) function or [virtual mode](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#virtualModeEnabled) to load data on demand, you should not add the search bar to prevent partial results. 
```

Comment:
What do you think about imperative mood here?

```suggestion
Note that only loaded nodes appear in the search results. Do not enable search if the TreeView uses the [createChildren](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#createChildren) function or [virtual mode](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#virtualModeEnabled) since the search will produce partial results.
```